src/output/tts/tts_online.py

```diff
@@ -19,6 +19,6 @@ def speak(self, text):
         sound.seek(0)
         pygame.mixer.music.load(sound, "mp3")
         pygame.mixer.music.play()
-        while pygame.mixer.music.get_busy():
-            pygame.time.delay(10)
-            pygame.event.poll()
+        for event in pygame.event.get():
+            if event.type == pygame.QUIT:
+                pygame.quit()
```

Comment:
this shouldn't be necessary. according to [the `pygame.quit` documentation](https://www.pygame.org/docs/ref/pygame.html#pygame.quit):

> [..] When the Python interpreter shuts down, this method is called regardless, so your program should not need it, except when it wants to terminate its pygame resources and continue. [..]
> Note: Calling `pygame.quit()` will not exit your program. Consider letting your program end in the same way a normal Python program will end.

so just a `break` here should be enough